Packs/Base/Scripts/CommonServerPython/CommonServerPython.py

```diff
@@ -1774,7 +1774,8 @@ def create_clickable_url(url):
     return '[{}]({})'.format(url, url)
 
 
-def tableToMarkdown(name, t, headers=None, headerTransform=None, removeNull=False, metadata=None, url_keys=None):
+def tableToMarkdown(name, t, headers=None, headerTransform=None, removeNull=False, metadata=None, url_keys=None,
+                    date_fields=None):
```

Comment:
please add the arg description in the docstring

Reply:
done.